library/alloc/src/collections/binary_heap.rs

```diff
@@ -870,7 +934,8 @@ impl<T> BinaryHeap<T> {
     /// The capacity will remain at least as large as both the length
     /// and the supplied value.
     ///
-    /// If the current capacity is less than the lower limit, this is a no-op.
+    /// Panics if the current capacity is smaller than the supplied
+    /// minimum capacity.
```

Comment:
I think this was probably a rebase error? I recall recently r+ing a PR that changed this to be a no-op.

Reply:
Yeah, that's a blunder